src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.cs

```diff
@@ -58,6 +60,8 @@ internal partial class CurlHandler : HttpMessageHandler
         private DecompressionMethods _automaticDecompression = DecompressionMethods.GZip | DecompressionMethods.Deflate;
         private SafeCurlMultiHandle _multiHandle;
         private GCHandle _multiHandlePtr = new GCHandle();
+        private bool _preAuthenticate = false;
```

Comment:
I assume the default for _preAuthenticate will move to the common defaults work once it's merged? https://github.com/dotnet/corefx/pull/2810